08-interop/README.md

```diff
@@ -281,3 +281,188 @@ guide on Zephyr's documentation.
    dst_l2addr: 32:A9:FA:65:10:6B:11:14
    ~~ PKT    -  4 snips, total size:  85 byte
    ```
+   
+Task #12 - GNRC Border Router with WiFi uplink to the Internet
+========================
```

Comment:
Ah one minor sure (since I'm not sure the issue generator for the RC tracking issues accounts for that or not): please extend the underline to the end of the upper line for each task.